### ISSUE TITLE
Analyze Debug Logs on disk without connecting to an org

### ISSUE THREAD

**Issue by cparmenter** [FuseInfoTech/FuseITSFDCExplorer#1]
Can you make it possible to analyze Apex debug logs on disk without connecting to a Salesforce org with FuseIT SFDC Explorer?

Thanks very much!

**Comment by FishOfPrey** [comment]
Thanks for the suggestion. How would you be looking at opening the log from disk?

I'm wondering if, in addition to the standard controls like opening it from a folder or pasting it in from the clipboard if the exe should accept a piped in log. Then it could be used on the command line with something like the SFDX CLI to bring in specific logs.

**Comment by cparmenter** [comment]
Good idea @FishOfPrey .  Command-line parameters might be a good way to go also.  /debugLogFile:<filePath> something like that....

Starting from the existing UI, I was thinking something more like add an "Open Debug Log" button or menu to the login screen. 
 Probably a menu command is the way to go.  Perhaps a new menu entry like "Tasks" or "Tools" with an "Open Debug Log from Disk" menu button below that main entry.

Or allow cancel on the login dialog and if cancelled, takes one to an empty UI with no org connected.  One could then go to the logs tab/pane and then load a log from disk.  Something like that.

Thanks very much for considering this.  It would be very helpful.
